_posts/2024-07-24-release.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Release 2024.30
+author: Priya
+date: 2024-07-24
+category: Jekyll
+layout: post
+---
+### Enhancements:
+* Combine Dataset: Updated the default setting for "Delete the dataset after combining" in the "Add to dataset" section to prevent unintended deletions and improve data retention.
+* Improved the email message for Google link in self-serve sign-up.
```

Comment:
Could we say "Improved the email message for linking Google account with Mammoth" and drop the self-serve sign-up